backend/src/services/release.ts

```diff
@@ -237,9 +239,11 @@ export async function newReleaseComment(user: UserInterface, modelId: string, se
 export async function getModelReleases(
   user: UserInterface,
   modelId: string,
+  q?: string,
```

Comment:
I'm never a fan of single character variable names like this. It's more confusing that simply giving it a proper name.

Consider calling this `querySemver` as you have later on.

Reply:
True. Fixed